### PR TITLE
meetup 2024-09-11

### PR DESCRIPTION
**Pull request by breml**

### CODE REVIEW

File: 2024-09-11/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Talk with code examples on the range-over function iterators (rangefuncs). We learn what they are, how they can be used and when they will be available.
+
+Links: [Slides](rangefuncs_-_iterators-in-Go_-_Lucas-Bremgartner.pdf) | [Code](https://github.com/breml/baerner-go-rangefuncs)  
+Speaker: [Github](https://github.com/breml) | [LinkedIn](https://www.linkedin.com/in/lucas-bremgartner-96b477/)
+
+## CPU limits & Go – Thomas Gosteli
+
+Lightning talk with demo on how CPU limits affect your Go application in containerized environments.
+
+Links: [Slides](CPU_limits_and_Go_-_Thomas-Gosteli.pdf) | [Code](https://github.com/ghouscht/cpu-limits-and-go)
```
Comment (pfiaux) [inline 2024-09-11/README.md:23]: The PDFs are still pending right?

Comment (breml) [reply 2024-09-11/README.md:23]: Jep, emails are out already.

File: 2024-09-11/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Lightning talk with demo on how CPU limits affect your Go application in containerized environments.
+
+Links: [Slides](CPU_limits_and_Go_-_Thomas-Gosteli.pdf) | [Code](https://github.com/ghouscht/cpu-limits-and-go)  
+Speaker: [Github](https://github.com/ghouscht) | [LinkedIn](https://www.linkedin.com/in/thomas-gosteli/)
+
+## ackbar – Mathis Kretz
+
+Mathis from bespinian about ackbar a tool to coordinate partitioned data streams to stateful workloads.
+
+Links: [Slides](ackbar_-_Mathis-Kretz.pdf) | [Code](https://github.com/bespinian/ackbar)
```
Comment (mkretz) [inline 2024-09-11/README.md:30]: Here are my slides in pdf. Does this work for you?
[ackbar - coordinating stateful computations.pdf](https://github.com/user-attachments/files/17017839/ackbar.-.coordinating.stateful.computations.pdf)

**Comment by breml** [comment]
@pfiaux , @philipsahli , @cldmstr please have a look, now the PR is complete. Thanks.

**Review by pfiaux — APPROVED** [review]

**Review by cldmstr — APPROVED** [review]
LGTM

**Pull request merged**